cds/modules/deposit/static/templates/cds_deposit/types/project/form.html

```diff
@@ -147,9 +147,7 @@ <h3 class="mb-5 mt-0">
                     </label>
                     <ui-select
                      multiple
-                     tagging
```

Comment:
So that we don't allow new items to be created, only the ones coming from the autocomplete